src/presence_analyzer/static/js/mean_time.js

```diff
@@ -27,19 +26,22 @@ function parseInterval(value) {
         $('#user_id').change(function() {
             var $selectedUser = $('#user_id').val(),
                 $chartDiv = $("#chart_div");
-            if($selectedUser === '') {
-                $("#avatar").hide();
-                $chartDiv.hide();
 
-            } else {
+            $.getJSON('api/v1/get_avatar/' + $selectedUser, function(result) {
+                $('#avatar').attr('src', result['avatar']);
+            });
+
+            if($selectedUser) {
                 $loading.show();
                 $chartDiv.hide();
 
-                $.getJSON('api/v1/get_avatar/' + $selectedUser, function(result) {
-                    $('#avatar').attr('src', result['avatar']);
-                });
-
                 $.getJSON('/api/v1/mean_time_weekday/' + $selectedUser, function(result) {
+                    if(result[404]) {
```

Comment:
Never do it like that. API should return 404 status with an appropriate message. Check if its 404 and then pass the msg.